src/mongocxx/private/index_view.hh

```diff
@@ -16,9 +16,14 @@
 
 #include <vector>
 
+#include <bsoncxx/builder/basic/array.hpp>
```

Comment:
After removing the needless headers from the public index_view header, more were required in the private impl.